android/app/build.gradle

```diff
@@ -53,12 +53,12 @@ android {
     }
 
     signingConfigs {
-        debug {
-            storeFile file('keystore.jks')
-            storePassword 'tech.pylons'
-            keyAlias 'key_tech_pylon'
-            keyPassword 'tech.pylons'
-        }
+//        debug  {
```

Comment:
Is it necessary to keep these debug statements?

Reply:
yes sir. We need it when we release builds.